build/ci/tools/osuosl/publish.sh

```diff
@@ -72,33 +80,20 @@ echo $PUBLISH_URL > $ARTIFACTS_DIR/env/publish_url.env
 cat $ARTIFACTS_DIR/env/publish_url.env
 
 # Delete old files
-if [ "$BUILD_MODE" == "nightly_build" ]; then 
+if [ "$BUILD_MODE" == "nightly_build" ]; then
     echo "Delete old MuseScoreNightly files"
 
-    if [ "$OS" == "linux" ]; then 
-        ssh -i $SSH_KEY musescore-nightlies@ftp-osl.osuosl.org "cd ~/ftp/$FTP_PATH; ls MuseScoreNightly*.AppImage       -t | tail -n +41 | xargs rm -f"
-        ssh -i $SSH_KEY musescore-nightlies@ftp-osl.osuosl.org "cd ~/ftp/$FTP_PATH; ls MuseScoreNightly*.AppImage.zsync -t | tail -n +41 | xargs rm -f"
-    else 
-        ssh -i $SSH_KEY musescore-nightlies@ftp-osl.osuosl.org "cd ~/ftp/$FTP_PATH; ls MuseScoreNightly* -t | tail -n +41 | xargs rm -f"
-    fi
+    ssh -i $SSH_KEY musescore-nightlies@ftp-osl.osuosl.org "cd ~/ftp/$FTP_PATH; ls MuseScoreNightly* -t | tail -n +42 | xargs rm -f"
```

Comment:
I think it's not important, but if we don't do a separate logic for Linux, there will be 2 times less available files in the history of days.

Reply:
@igorkorsukov, the latest zsync is a real file now, not a symlink, and I'm not keeping historic zsync files. So there will only be one less nightly on Linux, not half.

Reply:
I got it!